android/app/src/main/java/com/microsoft/codepush/react/CodePush.java

```diff
@@ -299,11 +312,10 @@ public static void setUsingTestConfiguration(boolean shouldUseTestConfiguration)
         testConfigurationFlag = shouldUseTestConfiguration;
     }
 
-    public void clearTestUpdates() {
-        if (isUsingTestConfiguration()) {
-            codePushPackage.clearTestUpdates();
-            removePendingUpdate();
-        }
+    public void clearUpdates() {
+        codePushPackage.clearUpdates();
```

Comment:
Since getBundleUrl is always called, even in debug mode, wouldn't this get called and cause some potential oddities in the dev workflow?